src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -690,3 +691,20 @@ def _mkdir_p(path):
             pass
         else:
             raise
+
+
+def update_acs(resource_group_name, container_service_name, new_agent_count):
+    from azure.cli.command_modules.acs.commands import cf_acs
+    client = cf_acs(None)
+    instance = client.get(resource_group_name, container_service_name)
+    instance.agent_pool_profiles[0].count = new_agent_count  # pylint: disable=no-member
+    return client.create_or_update(resource_group_name, container_service_name, instance)
+
+
+def list_container_services(resource_group_name=None):
+    ''' List Container Services. '''
+    from azure.cli.command_modules.acs.commands import cf_acs
+    client = cf_acs(None)
```

Comment:
In commands.py, add `cf_acs` to the command registration then the first argument to this method can be `client`.
This was the case when these commands were in the vm module.
https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py#L176

The same can be done to `update_acs` above.